examples/nek5000/README.md

```diff
@@ -0,0 +1,39 @@
+## Running Nek5000 examples
+
+### Generate meshes (boxes)
+
+You can generate box geometries using `generate-boxes.sh` script. Syntax
+for using the script is the following.
+```sh
+   generate_boxes log_2(<min_elem>) log_2(<max_elem>)."
+```
+
+Example:
+```sh
+./generate-boxes 2 4
+```
+This will generate three boxes with 4(=2^2), 8 and 16(=2^4) elements inside
+`boxes/b*` directories.
+
+### Make Nek5000 examples
+
+You can make Nek5000 examples by invoking `make-nek-examples.sh` script.
+```sh
+./make-nek-examples.sh
+```
+
+You can set the path to Nek5000 installation directory and the example
+names you want to be built by editing this script.
+
+### Run Nek5000 examples
+
+You can run Nek5000 examples by invoking `make-nek-examples.sh` script.
+Syntax for the command is 
+```sh
+  ./run-nek-example <example_name> <#mpi_ranks> <rea_name> <rea_and_map_path>"
+```
+
+Example:
+```
+  echo "Example ./run-nek-example ex1 4 b10 ./boxes/b10"
```

Comment:
Let's make the example something a user can run with the boxes generated above. Also perhaps name the Nek5000 tools that need to be built (`genbox`, `genmap`, and `reatore2`).

I tried running `./run-nek-example.sh ex1 2 b2 boxes/b2` and get 

```
[...]
 set initial conditions
 nekuic (1) for ifld            1
 call nekuic for vel
 xyz min     0.0000       0.0000       0.0000
 uvwpt min   0.0000       0.0000       0.0000       0.0000       0.0000
 PS min     0.99000E+22
 xyz max     1.0000       1.0000       1.0000
 uvwpt max   0.0000       0.0000       0.0000       0.0000       0.0000
 PS max    -0.99000E+22
 done :: set initial conditions

 call userchk
GFM max:  3.5084E-02  3.5084E-02  8.7711E-03  2.5577E-16  8.2593E-17  8.2593E-17
  8        4  3.4780E-12  3.4780E-12  3.4780E-12  error a        9
  8        4  3.4780E-12  3.4780E-12  3.4780E-12  error b        9
  8        4  3.4780E-12  3.4780E-12  3.4780E-12  error c        9
case scalar:      2  7      2         4          1372       686   9  4.03041E-03  3.40412E+05  4.47823E-04  6.52803E-07
EXIT: quit after cggos          7

an error occured: dying ...


total elapsed time             :   3.68236E-02 sec
total solver time w/o IO       :   0.00000E+00 sec
time/timestep                  :   0.00000E+00 sec
avg throughput per timestep    :   0.00000E+00 gridpts/CPUs
total max memory usage         :   1.20783E-01 GB

-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
-------------------------------------------------------
```

Do you know why?

Reply:
Once we get our timing for BP1,we call `exitti` subroutine which exit with a non-zero value. That is why the error. It is not due to an error in the program itself. I can remove this call as it is confusing.

Reply:
I will update the `run-nek-example` documentation as well.